src/bin/cli.ml

```diff
@@ -0,0 +1,126 @@
+open Cmdliner
+open Lang
```

Comment:
I think that it's cool if we add a line break between the opens and the let declarations